plugin-compass/src/main/java/com/mapbox/maps/plugin/compass/CompassViewPlugin.kt

```diff
@@ -183,7 +185,9 @@ open class CompassViewPlugin(
     bearing: Double,
     padding: Array<Double>
   ) {
-    update(bearing)
+    Handler(Looper.getMainLooper()).post {
```

Comment:
it is expected that camera callbacks are happening on a non ui thread?

Reply:
maybe reuse Handler instance?

Reply:
yes, main idea here is that they must return immediately from core and not get put to some scheduler. This call from render thread happens because `setSize` from renderer also calls `setCamera` under the hood. In all other cases I guess it will triggered from main thread.

Reply:
maybe setSize shouldn't happen on render thread?

Reply:
I think it should as it is connected with textures and surfaces being ready. Even if we move it some other thread as said above camera changes are not scheduled on main thread under the hood meaning that if user will call `setCamera` from non-main thread - we will crash in compass.